google/datalab/bigquery/commands/_bigquery.py

```diff
@@ -29,75 +29,81 @@
 import json
 import re
 
-import google.datalab.bigquery
-from google.datalab.bigquery._query_output import QueryOutput
-from google.datalab.bigquery._sampling import Sampling
+import google.datalab.bigquery as bigquery
+import google.datalab.contrib.bigquery.commands._bigquery as contrib_bq
 import google.datalab.data
 import google.datalab.utils
 import google.datalab.utils.commands
 
-BIGQUERY_DATATYPES = ['STRING', 'BYTES', 'INTEGER', 'INT64', 'FLOAT', 'FLOAT64', 'BOOLEAN',
-                      'BOOL', 'TIMESTAMP', 'DATE', 'TIME', 'DATETIME', 'RECORD']
-BIGQUERY_DATATYPES_LOWER = [t.lower() for t in BIGQUERY_DATATYPES]
-BIGQUERY_MODES = ['NULLABLE', 'REQUIRED', 'REPEATED']
-BIGQUERY_MODES_LOWER = [m.lower() for m in BIGQUERY_MODES]
-
-table_schema_schema = {
-  'definitions': {
-    'field': {
-      'title': 'field',
-      'type': 'object',
-      'properties': {
-        'name': {'type': 'string'},
-        'type': {'type': 'string', 'enum': BIGQUERY_DATATYPES + BIGQUERY_DATATYPES_LOWER},
-        'mode': {'type': 'string', 'enum': BIGQUERY_MODES + BIGQUERY_MODES_LOWER},
-        'description': {'type': 'string'},
-        'fields': {
-          'type': 'array',
-          'items': {
-            'allOf': [{'$ref': '#/definitions/field'}]
+from google.datalab.bigquery._query_output import QueryOutput
+from google.datalab.bigquery._sampling import Sampling
+
+
+class BigQuerySchema(object):
+  """A container class for commonly used BQ-related constants."""
+
+  BIGQUERY_DATATYPES = ['STRING', 'BYTES', 'INTEGER', 'INT64', 'FLOAT', 'FLOAT64', 'BOOLEAN',
```

Comment:
BIGQUERY_ prefix is redundant in BigQuerySchema.

Reply:
Done.